app/models/concerns/ats/workable/parse_url.rb

```diff
@@ -3,8 +3,8 @@ module Workable
     module ParseUrl
       def parse_url(url, _saved_ids = nil)
         regex_formats = [
-          %r{https://apply\.workable\.com/([^/]+)(?:/j/([^/]+))?},
-          %r{https://apply\.workable\.com/api/v1/accounts/([^/]+)(?:/jobs/([^/?]+)(?:\?.*)?)?}
+          %r{^https://apply\.workable\.com/(?:([^/]+)/)?j/([^/]+)},
```

Comment:
Impact of changes here? Does this also handle HTTP?

Reply:
This should prevent the parser bug that made CompanyCreator try to create Workable companies named 'j'. It only handles https, not http (none of the parsing methods do).